README.md

```diff
@@ -291,6 +284,71 @@ One other tip... This totally works:
 </glamorous.Div>
 ```
 
+#### Overriding component styles
+
+The most common scenario for using props is to override the style of an
+existing component (generated by `glamorous` or not). In this section we'll
+explain how you can use `className`, `css` and composition to override the
+styles of a component. Let's see how that can be done in the examples below.
+
+> Try this out in your browser [here](https://codesandbox.io/s/Kro5369wG)!
```

Comment:
Also, because this is what we're showing that people _should_ be doing, could you change this:

```jsx
// Simulates existing component
const UnstyledText = ({ className, children }) => (
  <div className={className}>{children}</div>
);
```

To this:

```jsx
// Simulates existing component
// Note that the most important parts of this component
// is that it will forward the children and className props
// and it will also forward any other valid div attributes
// like `id` and `data-`/`aria-` attributes
const UnstyledText = (props) => (
  <div {...props} />
);
```

Maybe that's more confusing than anything... I'm going to leave that up to you. But I want to avoid people asking us the question: "How do I make it so I can pass an `id` to my underlying component" which is why I brought it up. Maybe an additional comment would be sufficient...

Could also just do:

```jsx
// Simulates existing component
const UnstyledText = ({ className, children, ...rest }) => (
  <div className={className} {...rest}>{children}</div>
);
```

That's more work technically, but it at least indicates the importance of `className` and `children`. Your call :)